backends/bmv2/CMakeLists.txt

```diff
@@ -122,7 +149,8 @@ set (XFAIL_TESTS
   )
 
 if (HAVE_SIMPLE_SWITCH)
-  p4c_add_tests("bmv2" ${BMV2_DRIVER} "${BMV2_TEST_SUITES}" "${XFAIL_TESTS}")
+  p4c_add_tests("bmv2" ${BMV2_DRIVER} "${BMV2_V1MODEL_TEST_SUITES}" "${XFAIL_TESTS}")
+  #p4c_add_tests("bmv2" ${BMV2_DRIVER} "${BMV2_PSA_TEST_SUITES}" "${XFAIL_TESTS}" ${testExtraFlags})
```

Comment:
I think we need to have a HAVE_PSA_SWITCH macro defined which finds the `portable_switch` binary (or  `psa_switch` if renamed)